src/test/Main.hs

```diff
@@ -1,3 +1,5 @@
+{-# OPTIONS_GHC -Wno-error=overflowed-literals #-}
```

Comment:
I don't love this, but it was the easiest way to get everything working on both architectures.